### ISSUE TITLE
Audio playing icon to the right of tab instead of above webpage icon? (Photon style)

### ISSUE THREAD

**Issue by fhtdtdj** [black7375/Firefox-UI-Fix#166]
Thank you for your highly needed fix. Besides the padding and tabs, my biggest issue with Proton is this:

Using Lepton Photon Style, the icon for audio playing tabs is improved (on top of the page icon) compared to awful Proton new design (replacing the page icon - what were they thinking?), but still inferior to original Photon (to the right of the tab). See pic for what I mean. I became used to selecting or dragging tabs by clicking on the left side rather the right side to avoid clicking the closing button; now as a result when clicking on a tab playing audio to simply open it, it's easy to click on the disable audio function instead.

![2021-08-14 at 03 50 AM 001](https://user-images.githubusercontent.com/88416370/129463246-8424b89c-88ef-4333-bc5c-1b485785c7cc.JPG)

Is there a way to get the original location on the right back?

Research on this appears to indicate this is problematic due to Proton not allowing this to be changed in the userChrome CSS and requiring extra hacks of other settings files? I didn't see this issue mentioned on your pages? What do you think?

**Comment by black7375** [comment]
tldr; There are currently no plans.

That's why I placed it as above.
1. **Consistency**
  -  In general, picture-in-picture mode, pinned tab, etc., they are all located in the same place.
  - If the positions are the same, the muscle memory makes it easier to click.
  - Tab-related information, such as loading, faviton and container indicators, all in one place.
2. **More Text**
  - It's half-overlapped with Fabicon,  it takes up less space.
3. **Reduce Mistakes**
  - It can significantly reduce the chance of clicking `close` button by mistake.

---

Thanks for the suggestion, I'll add it to the tips wiki later when I have time.
